c_src/membrane_opus_plugin/_generated/nif/encoder.cpp

```diff
@@ -0,0 +1,151 @@
+#include "encoder.h"
+
```

Comment:
should this all bit in `.gitignore`, or should it not have been created at all?

Reply:
As above, remove it :slightly_smiling_face: